tests/models/graph_tests.py

```diff
@@ -1071,24 +1072,130 @@ def test_appending_a_branch_with_an_invalid_ontology_class(self):
         with self.assertRaises(GraphValidationError) as cm:
             graph.save()
 
-    def test_appending_to_published_graph(self):
-        graph = Graph.objects.get(node=self.rootNode)
-        admin = User.objects.get(username="admin")
-        graph.publish(user=admin)
-        self.addCleanup(graph.unpublish)
-
-        with self.assertRaises(GraphValidationError) as cm:
-            graph.append_node()
-        self.assertEqual(cm.exception.code, 1012)
-        with self.assertRaises(GraphValidationError) as cm:
-            graph.append_branch("http://www.nasa.gov/", graphid=self.NODE_NODETYPE_GRAPHID)
-        self.assertEqual(cm.exception.code, 1012)
-
-    def test_appending_published_branch_to_unpublished_graph(self):
-        graph = Graph.objects.get(node=self.rootNode)
-        admin = User.objects.get(username="admin")
-        branch = Graph.objects.get(graphid=self.NODE_NODETYPE_GRAPHID)
-        branch.publish(user=admin)
-        self.addCleanup(branch.unpublish)
-
-        graph.append_branch("http://www.nasa.gov/", graphid=self.NODE_NODETYPE_GRAPHID)
```

Comment:
Are these tests being deleted because the sense of what they are doing is captured in the tests that are being added?

Reply:
Nah they're being deleted for relevance. It's not longer possible to append a node to a publish graph, and there's no such thing as an unpublished graph after this feature